client/database.go

```diff
@@ -74,77 +72,68 @@ func (d *database) Transact(
 	return res, nil
 }
 
-func (d *database) ApplySchemasFromDirectory(ctx context.Context, path string) error {
-	schemas, err := readSchemaDir(path)
+func (db *database) ApplySchemasFromDirectory(ctx context.Context, dbName string, path string) error {
```

Comment:
do we need this method now?

Reply:
I think it can be useful helper in the future.